src/heretic/main.py

```diff
@@ -49,6 +50,76 @@
 )
 
 
+def obtain_merge_strategy(settings: Settings) -> str:
```

Comment:
```suggestion
def obtain_merge_strategy(settings: Settings) -> str | None:
```

Matches the actual behavior.